src/modules/logger/logger.cpp

```diff
@@ -606,6 +606,7 @@ void Logger::add_default_topics()
 	add_topic("actuator_controls_0", 100);
 	add_topic("actuator_controls_1", 100);
 	add_topic("actuator_outputs", 100);
+	add_topic("actuator_armed", 100);
```

Comment:
Can we please use `vehicle_status` instead? It already contains the arming state and a failsafe flag.
And it should be an `uint8_t` bitmask so that we can store the failsafe triggering reason (roll/pitch/altitude).

Reply:
Yes, I'll remove the logging of this topic. However, I discussed with @LorenzMeier yesterday and he would like to have a new "events" message that would contain a specific ID per event (ekf init error, parachute triggering, roll failure, ...), the level of severity (e.g.: info, warn, minor, major, critical, fatal; granularity TBD) and a few other parameters. This message will directly be published every time an event occurs.

Reply:
I agree we'll need that, but the whole events interface will be a bigger change.